platform/windows/display_server_windows.cpp

```diff
@@ -914,6 +915,38 @@ String DisplayServerWindows::clipboard_get() const {
 				GlobalUnlock(mem);
 			}
 		}
+
+	} else if (IsClipboardFormatAvailable(CF_HDROP)) {
+		HGLOBAL mem = GetClipboardData(CF_HDROP);
+		if (mem != nullptr) {
+			HDROP hDropInfo = (HDROP)GlobalLock(mem);
+			if (hDropInfo != nullptr) {
+
```

Comment:
Remove this blank line to fix CI:

```suggestion
```

Reply:
already have, i was just writing documentation. I don't know why clang-format isn't working for me but at least the CI catches it